src/outputFunctions.f90

```diff
@@ -103,18 +103,18 @@ end subroutine outputSolverParameters
   ! Write parameters used in calculation of photolysis rates to file.
   subroutine outputPhotoRateCalcParameters( t )
     use types_mod
-    use zenith_data_mod, only : latitude, longitude, secx, cosx, lha, sinld, cosld, theta, eqtime
+    use zenith_data_mod, only : latitude, longitude, secx, cosx, lha, sinld, cosld, eqtime
     implicit none
 
     real(kind=DP), intent(in) :: t
     logical :: first_time = .true.
 
     if ( first_time .eqv. .true. ) then
-      write (59, '(100A15) ') 't', 'LATITUDE', 'LONGITUDE', 'SECX', 'COSX', 'LHA', 'SINLD', 'COSLD', 'THETA', 'EQTIME'
+      write (59, '(100A15) ') 't', 'latitude', 'longitude', 'secx', 'cosx', 'lha', 'sinld', 'cosld', 'eqtime'
       first_time = .false.
     end if
 
-    write (59, '(100 (ES15.6E3)) ') t, latitude, longitude, secx, cosx, lha, sinld, cosld, theta, eqtime
+    write (59, '(100(1P e15.7)) ') t, latitude, longitude, secx, cosx, lha, sinld, cosld, eqtime
```

Comment:
Could you change the print format back to ES15.6E3 please? This is a regression from the rebase.

Reply:
I was afraid it may happen. Can you spot any other instance where I undid something or should I go back and re-rebase the whole lot?

Reply:
No, that looks like the only one. Unfortunately in the meantime I have merged another PR and made yours conflict again, sorry.